evm_loader/test_event.py

```diff
@@ -66,6 +102,45 @@ def call_signed(self, input):
         return http_client.send_transaction(trx, self.acc,
                                      opts=TxOpts(skip_confirmation=False, preflight_commitment="root"))["result"]
 
+    def create_storage_account(self, seed):
+        storage = PublicKey(sha256(bytes(self.acc.public_key()) + bytes(seed, 'utf8') + bytes(PublicKey(evm_loader_id))).digest())
+        print("Storage", storage)
+
+        if getBalance(storage) == 0:
+            trx = Transaction()
+            trx.add(createAccountWithSeed(self.acc.public_key(), self.acc.public_key(), seed, 10**9, 128*1024, PublicKey(evm_loader_id)))
+            result = http_client.send_transaction(trx, self.acc, opts=TxOpts(skip_confirmation=False))
+            print(result)
+
+        return storage
+
+    def call_partial_signed(self, input):
+        tx = {'to': solana2ether(self.reId), 'value': 1, 'gas': 1, 'gasPrice': 1,
+            'nonce': getTransactionCount(http_client, self.caller), 'data': input, 'chainId': 111}
+
+        (from_addr, sign, msg) = make_instruction_data_from_tx(tx, self.acc.secret_key())
+        assert (from_addr == self.caller_ether)
+        instruction = from_addr + sign + msg
+
+        storage = self.create_storage_account("1234")
+
+        trx = Transaction()
+        trx.add(self.sol_instr_keccak(make_keccak_instruction_data(1, len(msg), 9)))
+        trx.add(self.sol_instr_09_partial_call(storage, 3, instruction))
+        result1 = http_client.send_transaction(trx, self.acc, opts=TxOpts(skip_confirmation=False, preflight_commitment="root"))["result"]
+
+        trx = Transaction()
+        trx.add(self.sol_instr_keccak(make_keccak_instruction_data(1, len(msg), 9)))
```

Comment:
As I understand, keccak instruction doesn't need for continue and finalize instructions.

Reply:
Now it is needed for finalize.